hig.js

```diff
@@ -79,25 +81,40 @@ function runSpeech() {
 	const running = true;
 	var out = [];
 	genInterview(out);
-	while (speak(out)) {}
+	while (speak(out, os)) {}
 }
 
 function speak(phrases) {
```

Comment:
I don't understand how this is working you are calling the speak function with two arguments, but you haven't changed the function signature.

Reply:
Yeah, I overlooked this now that you mention it. Since it worked and did not give me an error I did not notice this problem. Ill make sure to double check next time and I will figure out why it did not cause an error and get back to you. Thanks!

Reply:
Apparently in javascript incorrectly formatted parameters are not enforced like other languages and do not cause errors.  Maybe when I declared the variable above the function was still able to reference it. 

An example and explanation of how javascript handles functions and parameters is shown here: https://stackoverflow.com/questions/11107823/what-happens-if-i-dont-pass-a-parameter-in-a-javascript-function

From the page above:

"Nothing will happen- meaning you won't get an error or a warning as passing the parameters in javascript is optional.
All the parameters that weren't "supplied" will have the undefined value."

Reply:
That being said, I fixed the signature so it passes the os. I could probably look into this more as perhaps I do not need to pass the os into the function everytime I call it if it can be referenced as a static variable.

Reply:
I went ahead and took "os" out of the function call all together as the speak function seems to be able to reference it as a global variable.  In my opinion this makes it much easier to read because A) I do not need to pass in the os every time I could the function and B.) it is not necessary because the operating system is not something that needs to be mutable.